packages/databricks-vscode/package.json

```diff
@@ -293,6 +299,10 @@
                     "command": "databricks.quickstart.open",
                     "when": "view == configurationView"
                 },
+                {
+                    "command": "databricks.bundle.initNewProject",
+                    "when": "view == configurationView"
+                },
```

Comment:
The command has no icon. How does it show up in the bar?

Reply:
It's grouped with the quick Quickstart

<img width="676" alt="Screenshot 2024-01-24 at 11 17 35" src="https://github.com/databricks/databricks-vscode/assets/148094031/c730c420-1d2d-4286-98f5-51b57db19d3f">